spec/services/search/classified_listing_spec.rb

```diff
@@ -100,4 +100,159 @@
       described_class.delete_index(index_name: other_name)
     end
   end
+
+  describe "::search_documents" do
+    let(:classified_listing1) { create(:classified_listing) }
+    let(:classified_listing2) { create(:classified_listing) }
+    let(:classified_listing3) { create(:classified_listing) }
+    let(:classified_listing4) { create(:classified_listing) }
+    let(:classified_listing5) { create(:classified_listing) }
```

Comment:
Since we only use 5 of these for 1 spec I think it would be better to make 5 for that one spec and then only make a single classfied listing here. Making 5 before every single spec when we are only going to use one for most is going to slow this down.